pkg/vex/vex.go

```diff
@@ -342,17 +397,20 @@ func (vexDoc *VEX) CanonicalHash() (string, error) {
 	cString := fmt.Sprintf("%d", vexDoc.Timestamp.Unix())
 
 	// 2. Document version
-	cString += fmt.Sprintf(":%s", vexDoc.Version)
+	cString += fmt.Sprintf(":%d", vexDoc.Version)
+
+	// 3. Author identity
+	cString += fmt.Sprintf(":%s", vexDoc.Author)
 
-	// 3. Sort the statements
+	// 4. Sort the statements
 	stmts := vexDoc.Statements
 	SortStatements(stmts, *vexDoc.Timestamp)
 
-	// 4. Now add the data from each statement
+	// 5. Now add the data from each statement
```

Comment:
Does this mean the `4a`, `4b`, etc. below should change, too?

Reply:
Good catch :+1: